src/Illuminate/Validation/Concerns/ValidatesAttributes.php

```diff
@@ -1344,6 +1344,10 @@ public function validateMaxDigits($attribute, $value, $parameters)
      */
     public function validateMimes($attribute, $value, $parameters)
     {
+        if(is_array($value)) {
+            return $this->checkMultiValues(__FUNCTION__, $attribute, $value, $parameters);
+        }
+
```

Comment:
Why were the array checks only added to the mime rules? 

What if someone wants to validate max file size, required, minimum file size?